src/containers/Trigger/Trigger.tsx

```diff
@@ -378,11 +385,18 @@ export const Trigger = () => {
           'If you are selecting end of the month dates, then for the ones not present i.e. 30, 31, the selection will default to the last day of that month.'
         ),
     },
+    {
+      component: TriggerType,
+      name: 'groupType',
+      handleOnChange: (value: any) => setGroupType(value),
+      groupType: groupType,
+      isWhatsAppGroupEnabled: isWhatsAppGroupEnabled,
+    },
     {
       component: AutoComplete,
       name: 'groupIds',
       label: t('Select collection'),
-      options: collections.groups,
+      options: collections?.groups,
```

Comment:
Collection options should  be filtered based on the type of Collection. RIght now it will load all the collections.